src/widgets/buttons.ts

```diff
@@ -724,6 +730,59 @@ export function renderAsRow (dom: HTMLDocument, pred: NamedNode, obj: NamedNode,
   return tr
 }
 
+/**
+ * A Div to represent a draggable person, etc in a list
```

Comment:
if this is meant to represent a "draggable", why can draggable be set to false?

Reply:
Good question, I'm not really sure this is the code from the original and I didn't want to change it too much.

Reply:
Maybe I should just make this one draggable only?

Reply:
I would say yes, make it dragable only

Reply:
Thank you, I did :)